doc/index.rst

```diff
@@ -522,7 +521,7 @@ Or define ``CCACHE_DIR`` per job.
 NOTE:
   * Beware, if you use `run_ci <https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#id39>`__, the files will be owned by root!
   * Caching may not work for packages with "smaller" number of files (see also `this discussion <https://github.com/ros-industrial/industrial_ci/pull/182>`__).
-
+  * In Gitlab CI, when using ``docker:dind``, ``CCACHE_DIR`` can only be set to a folder within ``/builds/$CI_PROJECT_PATH/shared`` (see also `this issue <https://github.com/ros-industrial/industrial_ci/issue/536>`__)
```

Comment:
With Gitlab CI cache should always be inside the project folder (https://docs.gitlab.com/ee/ci/yaml/README.html#cachepaths).
```YAML
variables:
  TMPDIR: "${CI_PROJECT_DIR}.tmp"
  CCACHE_DIR: ${CI_PROJECT_DIR}/ccache
  
cache:
  key: "${CI_JOB_NAME}"
  paths: 
    - ccache

```

Our  Gitlab config should be changes accordingly: https://gitlab.com/ipa-mdl/industrial_ci/-/blob/test-ccache/.gitlab-ci.yml

Reply:
Updated accordingly